Makefile

```diff
@@ -1,11 +1,11 @@
-openapi-generator-version:=6.0.1
+openapi-generator-version:=6.2.1
```

Comment:
Can we do this upgrade in a separate PR? Ideally, model changes should come out from our bot in a PR like #888. In this scope of this ticket, only `JSON` classes should change.